tonic/src/transport/server.rs

```diff
@@ -421,3 +511,27 @@ where
         })
     }
 }
+
+#[derive(Default, Clone, Debug)]
+#[doc(hidden)]
+pub struct Unimplemented;
```

Comment:
This should probably have a single private field to protect against users accidentally constructing it...